docs/features.md

```diff
@@ -0,0 +1,18 @@
+# Supported Kubernetes features
+
+This is an uncomplete list of Kubernetes features supported by rktlet.
+
+| Feature                | Supported | Comments |
+| ---------------------- |:---------:| -------- |
+| Pod lifecycle          | YES       |          |
+| Container lifecycle    | YES       |          |
+| Logging                | YES       |          |
+| `kubectl attach`       | NO        | See [#8](https://github.com/kubernetes-incubator/rktlet/issues/8) |
+| `kubectl port-forward` | NO        |          |
+| kubectl exec           | YES       |          |
```

Comment:
kubectl exec does not have the backticks like kubectl attach and port-forward. It should be consistent.